### DISCUSSION TITLE
Prusaslicer now natively supports post-processing

### DISCUSSION THREAD

**Discussion by Sonosus** [Sakati84/3DPrintColorizer#94] [Ideas]
From the latest release:
> G-Code substitutions
PrusaSlicer supports processing of the generated G-code with an external post-processor for a long time. Launching an external post-processor is versatile, however it is complex to setup, the post-processing scripts must be distributed with the project and the interpreter running the post-processing script (for example Python) has to be available. We collected many real world use-cases for the post-processing scripts and concluded that many of them could be covered by a simple "Find and Replace" tool with regular expression matching and substitutions.

>To make life easier for everyone who only needs basic post-processing, we now integrated such a tool directly into PrusaSlicer. It is accessible from Print Settings -> Output options and allows to add a number of find and replace pairs, with optional regex matching, case sensitivity and whole word matching, similar to what common text editors offer. The G-Code is processed before it is previewed, so your changes will be visible in the G-Code preview.

Source: https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.4.1-beta1

I wonder if it would be possible to integrate 3DPrintColorizer into this.

**Comment by xantrk** [comment]
Yeah Prusaslicer support would be amazing. After I switched to Prusaslicer from Cura I never looked back, it is so much better IMO.

**Reply by LouisLe18** [reply]
agreed

**Comment by LouisLe18** [comment]
yes this would be awesome +1000
